charts/external-dns/templates/deployment.yaml

```diff
@@ -109,9 +113,9 @@ spec:
             {{- toYaml .Values.livenessProbe | nindent 12 }}
           readinessProbe:
             {{- toYaml .Values.readinessProbe | nindent 12 }}
-          {{- if or .Values.secretConfiguration.enabled .Values.extraVolumeMounts }}
+          {{- if or .Values.secretConfiguration.mountPath .Values.extraVolumeMounts }}
           volumeMounts:
-            {{- if .Values.secretConfiguration.enabled }}
+            {{- if .Values.secretConfiguration.mountPath }}
```

Comment:
```suggestion
            {{- if .Values.secretConfiguration.enabled }}
```

Reply:
Should not unnecessarily mount secrets into the external-dns container if they're not needed there.

Reply:
ok for this one.